packages/flutter/lib/src/material/drawer.dart

```diff
@@ -8,6 +8,9 @@ import 'package:flutter/widgets.dart';
 import 'colors.dart';
 import 'list_tile.dart';
 import 'material.dart';
+import 'scaffold.dart';
+
+enum DrawerType { START, END }
```

Comment:
s/START/start/
s/END/end/

I wonder if `DrawerAlignment` would be a better name for this enum.  You could imagine `DrawerType` as something more general (e.g., floating, expanding, circular).

Reply:
Fixed